.github/workflows/report-server.yml

```diff
@@ -0,0 +1,60 @@
+name: Report Server Tests
+
+on:
+  push:
+    paths:
+      - 'server/**'
+      - '.github/workflows/**'
```

Comment:
I'll remove this in a follow on PR - it is needed so the tests run for this PR that setup the tests :)